### PR TITLE
Replace Label  Fortnightly to Every two weeks

### PR DESCRIPTION
**Pull request by Dharmraj-48**
Replace label as Every two week in Payroll Frequency
Screenshot from 2020-03-16 16-15-33

### CODE REVIEW

File: erpnext/hr/doctype/payroll_entry/payroll_entry.py [hunk 1]
```diff
@@ -443,7 +443,7 @@ def get_start_end_dates(payroll_frequency, start_date=None, company=None):
 def get_frequency_kwargs(frequency_name):
 	frequency_dict = {
 		'monthly': {'months': 1},
-		'fortnightly': {'days': 14},
+		'Every two weeks': {'days': 14},
```
Comment (Alchez) [inline erpnext/hr/doctype/payroll_entry/payroll_entry.py:446]: Make the key lowercase

File: erpnext/hr/doctype/payroll_entry/payroll_entry.py [hunk 2]
```diff
@@ -456,7 +456,7 @@ def get_end_date(start_date, frequency):
 	frequency = frequency.lower() if frequency else 'monthly'
 	kwargs = get_frequency_kwargs(frequency) if frequency != 'bimonthly' else get_frequency_kwargs('monthly')
 
-	# weekly, fortnightly and daily intervals have fixed days so no problems
+	# weekly, Every two weeks and daily intervals have fixed days so no problems
```
Comment (Alchez) [inline erpnext/hr/doctype/payroll_entry/payroll_entry.py:459]: Make the key lowercase

**Review by Alchez — CHANGES_REQUESTED** [review]
Everything else looks fine

**Comment by vishdha** [comment]
@Dharmraj-48  Please do changes for salary slip and payroll entry too

**Comment by Alchez** [comment]
Same as #348, closing

**Pull request closed without merging**